app/views/articles/_sidebar_additional.html.erb

```diff
@@ -27,6 +27,9 @@
           </div>
         </div>
       <% end %>
+
+      <%= render "articles/sidebar_campaign" if SiteConfig.campaign_sidebar_enabled %>
```

Comment:
Not that it matters much, but `SiteConfig` adds predicate methods for boolean values in case you prefer`SiteConfig.campaign_sidebar_enabled?`